src/scrapers/platformen/mijnbuurtje.py

```diff
@@ -0,0 +1,180 @@
+                                'transform': lambda elements: self.find_initiative_links(elements)}}
+
+                # initialize TreeParser using url and schemas, returns html tree
+                initiative_parser = TreeParser(list_page_url, None, schemas)
+                if initiative_parser.tree is None:
+                    break
+
+                output = initiative_parser.apply_schemas()
+                for uri in output['initiatives']:
+                    yield InitiativeImport(source_uri=uri)
+
+                page_counter = page_counter + 1
+        except Exception as ex:
+            raise ScrapeException("Error loading list of initiatives") from ex
+
+    def complete(self, initiative: InitiativeImport):
+        try:
+            session_metadata = self.item_parser.get_session_metadata(initiative.source_uri)
+            full_initiative = self.item_parser.apply_schemas(metadata=session_metadata,
+                                                             url=initiative.source_uri)
+            for key, value in full_initiative.items():
+                setattr(initiative, key, value)
+
+            if not initiative.location:
+                initiative.location = self.config.location
+        except Exception as ex:
+            raise ScrapeException(f"Error scraping {initiative.source_uri}") from ex
+
+    def find_initiative_links(self, elements):
+        """
+        filters links. Input seems to be 1 on 1 now. Extracted to method for easier debugging.
+        """
+        initiatives = []
+        for e in elements:
+            links = re.findall(self.config.details_endpoint + "\\d{4,5}", e.attrib['href'])
+            if len(links) > 0:
+                initiatives.append(links)
+
+        return initiatives
+
+    @staticmethod
+    def strip_text(element, strip: str):
+        if element is None:
+            return None
+        if not element.text:
+            return None
+
+        stripped = element.text.replace(strip, "")
+        return stripped
+
+    @staticmethod
+    def recursive_text(elements):
+        texts = []
+        for e in elements:
+            if type(e) in [etree._ElementUnicodeResult]:
+                texts.append(f"{e.strip()}")
+            else:
+                texts.append(f"{e.text.strip()}")
+        return ' '.join(texts)
+
+    @staticmethod
+    def format_group(original_group: str):
+        if len(re.findall("Hulpvraag.*?", original_group)) > 0:
+            return 'demand'
+        else:
+            return 'supply'
+
+    @staticmethod
+    def format_organizer(organizer):
+        if organizer is not None:
+            organizer = re.sub('-', ' ', organizer)
+            name_segments = organizer.split()
+            organizer_name = None if len(name_segments) == 0 else name_segments[0]
+
+            return organizer_name
+
+    # I assume there's a better way using locales
+    MONTHS = ["januari", "februari", "maart", "april", "mei", "juni", "juli",
```

Comment:
```
import locale
from calendar import month_name

locale.setlocale(locale.LC_ALL,'nl_NL.utf-8')
MONTHS = [month for month in month_name if month]
```

Dan moet je wel in je env `nl_NL.UTF-8` beschikbaar hebben

Reply:
Hoe is dat met de command line te doen? Want dan kan het wel ergens in een script geplaatst worden.

Aan de andere kant,nu de code er toch al is is kan ik het altijd nog als falll
back gebruiken als de locale er niet is. 😇